Assets/3D/bevy_third_person_camera.toml

```
@@ -0,0 +1,4 @@
+name = "bevy_third_person_camera"
+description = "A simple, plug & play third person camera for bevy."
+link = "https://crates.io/crates/bevy_third_person_camera"
+image = 'assets\assets\3d\bevy_third_person_camera.gif'
```

I think it needs to be like this:

```suggestion
image = "bevy_third_person_camera.gif"
```